src/robusta/core/sinks/robusta/robusta_sink.py

```diff
@@ -524,3 +534,58 @@ def __publish_new_namespaces(self, namespaces: List[NamespaceInfo]):
 
     def get_global_config(self) -> dict:
         return self.registry.get_global_config()
+
+    def __update_node(self, new_node: Node, operation: K8sOperationType):
```

Comment:
will this be called every time a new pod is added or CPU/mem allocation changes?

Reply:
do you have a way of checking how often this is called